charts/sn-platform/templates/openshift/scc.yaml

```diff
@@ -17,7 +17,7 @@
 # under the License.
 #
 
-{{- if and .Values.openshift.enabled .Values.openshift.ssc.enabled }}
+{{- if and .Values.openshift.enabled .Values.openshift.scc.enabled (not .Values.openshift.scc.name) }}
```

Comment:
How about adding a new flag `openshift.scc.create` to control the creation of the SCC?